.github/workflows/linux_conda.yml

```diff
@@ -0,0 +1,83 @@
+name: 'linux / conda'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  py310:
+    runs-on: ubuntu-20.04
+    name: 'linux conda 3.10'
+    defaults:
+      run:
+        shell: bash
+    env:
+      CONDA_ENV: 'environment.yml'
+      DISPLAY: ':99.0'
+      MNE_LOGGING_LEVEL: 'warning'
+      MKL_NUM_THREADS: '1'
+      PYTHONUNBUFFERED: '1'
+      PYTHON_VERSION: '3.10'
+    steps:
+      - name: 'checkout MNE-Python repo'
+        uses: actions/checkout@v3
+        with:
+          name: mne-tools/mne-python
+          ref: refs/heads/main
+      - name: 'Setup xvfb'
+        run: ./tools/setup_xvfb.sh
+      - name: 'Setup conda'
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: 'mne'
+          python-version: ${{ env.PYTHON_VERSION }}
+          environment-file: ${{ env.CONDA_ENV }}
+      - name: 'Install MNE-Python dependencies'
+        shell: bash -el {0}
+        run: |
+          ./tools/github_actions_dependencies.sh
+          source tools/get_minimal_commands.sh
+      - name: 'Check minimal commands'
+        shell: bash -el {0}
+        run: mne_surf2bem --version
+      - name: 'Install MNE-Python'
+        shell: bash -el {0}
+        run: ./tools/github_actions_install.sh
+      - name: 'Check Qt GL'
+        shell: bash -el {0}
+        run: |
+          QT_QPA_PLATFORM=xcb LIBGL_DEBUG=verbose LD_DEBUG=libs python -c "import pyvistaqt; pyvistaqt.BackgroundPlotter(show=True)"
+      - name: 'Show infos'
+        shell: bash -el {0}
+        run: ./tools/github_actions_infos.sh
+      - name: 'Get testing version'
+        shell: bash -el {0}
+        run: ./tools/get_testing_version.sh
+      - name: 'Cache testing data'
+        uses: actions/cache@v3
+        with:
+          key: ${{ env.TESTING_VERSION }}
+          path: ~/mne_data
+      - name: 'Download testing data'
+        shell: bash -el {0}
+        run: ./tools/github_actions_download.sh
+      - name: 'Print locale'
+        shell: bash -el {0}
+        run: ./tools/github_actions_locale.sh
+      - name: 'Clone MNE-Incubator'
+        uses: actions/checkout@v3
+        with:
+          name: mne-tools/mne-incubator
+          ref: refs/heads/main
+      - name: 'Run tests'
+        shell: bash -el {0}
+        run: ./tools/github_actions_test.sh
```

Comment:
this is unchanged, but IIUC it should (?) point to the script within `mne_incubator` instead of `mne_python`